### PR TITLE
Write up contributing post

### LINKED ISSUE #13: write up detailed contribution instructions
We should write up contributing instructions, including
 - how to clone, render locally, and submit a PR
 - how to create a post from an issue
 - our contributing guidelines

### PR DESCRIPTION
**Pull request by munkm**
This should resolve munkm/yt-blog#13 

@chrishavlin let me know if this is helpful!

### CODE REVIEW

**Review by matthewturk — APPROVED** [review]
Looks good!  I like that you separate out the two options.  Should we emphasize that Option 1 is a lot easier?

**Comment by munkm** [comment]
Good idea. I'll add a note for that!

File: content/posts/contributing.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+preview the website locally. Execute:
+
+``` bash
+$ hugo server -D
+```
+
+To create a local server of the page, served (by default) to localhost:1313
+
+The `-D` flag will show draft posts, so as you start writing your post it will
+show up on this locally served version of the webpage.
```
Comment (chrishavlin) [inline content/posts/contributing.md]: ah hah! wish I had known this... Glad you noted this here.

File: content/posts/contributing.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+Total in 32 ms
+```
+
+### Create a New Post
+
+To create a new post with the header information that will be used to attribute
+your authorship and tag the post, execute:
+
+``` bash
+$ hugo new posts/mycoolpost.md
```
Comment (chrishavlin) [inline content/posts/contributing.md:209]: related to the `hugo server -D` instructions above: `hugo new *` adds a `draft: true` attribute to the new file (or at least it did for me?). `hugo server -D` will show it locally, but does that line need to be deleted before submitting a PR?

**Review by chrishavlin — APPROVED** [review]
great! had one comment/question about the `draft:` attribute, but this captured everything I did offline for my initial posts!

File: content/posts/contributing.md (hunk 2 above)
Comment (munkm) [reply content/posts/contributing.md:209]: Great question! I will clarify this.

Comment (munkm) [reply content/posts/contributing.md:209]: Ok @chrishavlin I added some clarifying parts to the two sections you pointed out (both to explain -D and to talk about the draft line). Does this clarify enough?

**Comment by munkm** [comment]
@matthewturk I also mentioned option 1 was easier in the text I pushed.

**Comment by munkm** [comment]
I also realized that we are using `master` as a branchname here. When it moves into the org I suggest we make the default branch `main`.

**Pull request merged**